java/tests/com/google/privacy/differentialprivacy/ConfidenceIntervalGaussianTest.java

```diff
@@ -0,0 +1,289 @@
+package com.google.privacy.differentialprivacy;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+import static org.junit.Assert.assertThrows;
+
+import com.google.auto.value.AutoValue;
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import javax.annotation.Nullable;
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Enclosed.class)
+public class ConfidenceIntervalGaussianTest {
```

Comment:
nit: can we name this GaussianNoiseConfidenceIntervalTest, that way it will be next to GaussianNoiseTest when ordered alphabetically ;)

Also, we should probably test computeConfidenceInterval with sigma as a parameter since this is also part of the public facing API

Reply:
I think "computeConfidenceInterval with sigma as a parameter" should be probably be private because it doesn't have argument checking which makes sense because it is called as a common function from both interface functions.